update/factory/fullSyncInterceptors.go

```diff
@@ -246,7 +246,11 @@ func checkBaseParams(
 	if check.IfNil(cryptoComponents.BlockSigner()) {
 		return process.ErrNilSingleSigner
 	}
-	if check.IfNil(cryptoComponents.MultiSigner()) {
+	multiSigner, err := cryptoComponents.GetMultiSigner(0)
```

Comment:
is this ok?

Reply:
this one checks that we at least have a configured multisigner (for epoch 0)
we need minimum one multisigner.
If we have more, the change is that we will have epoch ranges where different multisigners are used, but the minimum required is to have at least one.

Reply:
Still doesn't feel right. We are using this only to test the next error. This will always work as we will always a component for the epoch 0 (otherwise we will not have the integration & system tests pass). Suggestion to completely remove L249-L255